int64 pixel_x
int64 pixel_y
float32 camera_pan
float32 camera_tilt
float32 height
---
float32 x
float32 y 